### ISSUE TITLE
Make sample data for grade inqueries

### ISSUE THREAD

**Issue by sbelsk** [Submitty/Submitty#6828]
**What problem are you trying to solve with Submitty**
Grade inquiries are not enabled by default in the sample course, and no sample data exists for them currently. Having it enabled with sample data will increase the coverage of the accessibility tests, help in developing, and can be a useful infrastructure for future e2e testing.

**Describe the way you'd like to solve this problem**
The course config file should be changed to enable grade inquiries when creating the sample course, and sample student/TAs data should be loaded to it automatically.
